docs/UserGuide.md

```diff
@@ -20,21 +20,49 @@ you can save it in the Personal Tracker provided by ExchangeCourseMapper!
 
 ## Features 
 
-{Give detailed description of each feature}
+### Adding a course mapping: `add`
+Adds a new course mapping into `myList.json` file for storage.Course mapping is subject to validation 
+to ensure that the course mapping is valid and that the university provided is an Oceania university. 
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+Format: `add NUS_COURSE_CODE /pu PARTNER_UNIVERSITY_NAME /coursepu PU_COURSE_CODE`
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+* All 3 parameters `NUS_COURSE_CODE`, `PARTNER_UNIVERISTY_NAME` and `PU_COURSE_CODE` are case-insensitive.
+* Do not add punctuation to the above three parameters
+* Indicate the full name of the partner university for `PARTNER_UNIVERISTY_NAME`. For example, indicate
+`The Australian National University` instead of `Australian National University` or `ANU`. 
 
```

Comment:
May need to review some merge conflicts

Reply:
Thanks for pointing it out! I have resolved the conflicts with the new commit